plugins/uefi-dbx/fu-plugin-uefi-dbx.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-efivar.h"
+#include "fu-hash.h"
+#include "fu-uefi-dbx-common.h"
+#include "fu-uefi-dbx-file.h"
+
+struct FuPluginData {
+	gchar			*fn;
+};
```

Comment:
This seems like overkill to keep a member just for something that you use in startup.

Is this because for HSI stuff later you want to have access to run at it outside of the startup function?

Reply:
> because for HSI stuff later

Exactly that.